package.json

```diff
@@ -10,12 +10,11 @@
   "scripts": {
     "db:seed": "cd backend && yarn run db:seed",
     "db:reset": "cd backend && yarn run db:reset",
-    "cypress:backend:server": "cd backend && yarn run test:before:server",
-    "cypress:backend:seeder": "cd backend && yarn run test:before:seeder",
-    "cypress:webapp": "cd webapp && cross-env GRAPHQL_URI=http://localhost:4123 yarn run dev",
-    "cypress:setup": "run-p cypress:backend:* cypress:webapp",
-    "cypress:run": "cypress run --browser chromium",
-    "cypress:open": "cypress open --browser chromium",
+    "cypress:backend": "cd backend && cross-env GRAPHQL_URI=http://localhost:4001 yarn run dev",
+    "cypress:webapp": "cd webapp && cross-env GRAPHQL_URI=http://localhost:4001 NUXT_PORT=3001 yarn run dev",
+    "cypress:setup": "run-p cypress:backend cypress:webapp",
+    "cypress:run": "cross-env CYPRESS_BASE_URL=http://localhost:3001 cypress run --browser chromium",
+    "cypress:open": "cross-env CYPRESS_BASE_URL=http://localhost:3001 cypress open --browser chromium",
```

Comment:
is it because of this refactor? hard to tell how this is related to form validation